znnl/models/flax_model.py

```diff
@@ -165,18 +153,69 @@ def _init_params(self, kernel_init: Callable = None, bias_init: Callable = None)
 
         return params
 
+    def _ntk_apply_fn(self, params, inputs: np.ndarray):
+        """
+        Return an NTK capable apply function.
+
+        Parameters
+        ----------
+        params: dict
+                Contains the model parameters to use for the model computation.
+                It is a dictionary of structure
+                {'params': params, 'batch_stats': batch_stats}
+        inputs : np.ndarray
+                Feature vector on which to apply the model.
+
+        TODO(Konsti): Make the apply function work with the batch_stats.
```

Comment:
What is the issue with it?

Reply:
Right now and also before the PR, there is no proper handling of batch statistics in the flax class. If you would want to have batch normalization within a model it would not work properly.
I want to fix that (in a different PR) to allow flax models to have batchstats.